<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="_tab">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Greibach Normálforma</string>
        </attribute>
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>450</y>
           <width>75</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Generál</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="greibach_next_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>450</y>
           <width>75</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Következő</string>
         </property>
         <property name="autoRepeatDelay">
          <number>300</number>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="greibach_back_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>450</y>
           <width>75</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Vissza</string>
         </property>
        </widget>
        <widget class="QLabel" name="gn_count">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>450</y>
           <width>66</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="left_rec_lab">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>100</y>
           <width>101</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Balrekurzió</string>
         </property>
        </widget>
        <widget class="QLabel" name="sort_lab">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>130</y>
           <width>111</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Sorbarendezés</string>
         </property>
        </widget>
        <widget class="QLabel" name="substitute_lab">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>160</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Behelyettesítés</string>
         </property>
        </widget>
        <widget class="QLabel" name="greibach_info_label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>571</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="n_lab">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>70</y>
           <width>201</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Nem-terminálisok bevezetése</string>
         </property>
        </widget>
        <widget class="QLabel" name="reduction_lab">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>190</y>
           <width>81</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Redukálás</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="greibach_t_edit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>451</width>
           <height>361</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="end_lab">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>220</y>
           <width>66</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Kész</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Early Algoritmus</string>
        </attribute>
        <widget class="QTextEdit" name="early_text">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>291</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="early_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>30</y>
           <width>98</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Indítás</string>
         </property>
        </widget>
        <widget class="QPushButton" name="early_next_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>30</y>
           <width>98</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Következő</string>
         </property>
        </widget>
        <widget class="QPushButton" name="early_back_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>30</y>
           <width>98</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Vissza</string>
         </property>
        </widget>
        <widget class="QLabel" name="early_progress_lab">
         <property name="geometry">
          <rect>
           <x>680</x>
           <y>30</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QTextEdit" name="early_info_text">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>641</width>
           <height>121</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="early_helper">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>631</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Adja meg a levezetendő szót a terminálisokat szóközzel elválasztva!</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Veremautomata</string>
        </attribute>
        <widget class="QTextEdit" name="pd_text">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>211</width>
           <height>311</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="pd_grammar">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>110</y>
           <width>421</width>
           <height>311</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="push_down_back_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>440</y>
           <width>98</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Vissza</string>
         </property>
        </widget>
        <widget class="QPushButton" name="push_down_next_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>440</y>
           <width>98</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Következő</string>
         </property>
        </widget>
        <widget class="QLabel" name="push_down_progress_lab">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>440</y>
           <width>66</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QTextEdit" name="pd_info">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>211</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="pd_progress_lab">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>440</y>
           <width>66</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="back_pd_grammar_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>440</y>
           <width>98</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Vissza</string>
         </property>
        </widget>
        <widget class="QPushButton" name="next_pd_grammar_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>440</y>
           <width>98</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Következő</string>
         </property>
        </widget>
        <widget class="QLabel" name="push_down_lab">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Veremautomata:</string>
         </property>
        </widget>
        <widget class="QLabel" name="pd_gammar_lab">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>50</y>
           <width>271</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>A veremautomatából generált nyelvtan:</string>
         </property>
        </widget>
        <widget class="QLabel" name="pd_grammar_progress_lab">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>440</y>
           <width>66</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QTextEdit" name="pd_gr_info">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>70</y>
           <width>421</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="generate_pushdown">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>98</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>generál</string>
         </property>
        </widget>
        <widget class="QPushButton" name="generate_grammar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>20</y>
           <width>98</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>generál</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fájl</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionNew_grammar"/>
    <addaction name="actionSave_grammar"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNew_grammar">
   <property name="text">
    <string>Új Nyelvtan...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Kilépés</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Nyelvtan betöltése...</string>
   </property>
  </action>
  <action name="actionSave_grammar">
   <property name="text">
    <string>Nyelvtan mentése...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
